src/main/resources/kroki-extension.rb

```diff
@@ -63,8 +64,16 @@ def process(processor, parent, attrs, diagram_type, diagram_text)
           'role' => role ? "#{role} kroki" : 'kroki',
           'target' => image_url,
           'alt' => target || 'diagram',
-          'title' => title
+          'title' => title,
```

Comment:
Apparently, the attribute `title` is not used so we should not set it:

```suggestion
```

Reply:
I just noticed that the `title` was already present... reading the code of Asciidoctor I think it's unused but without tests it's really hard to say. Maybe it's better to leave it for now.